src/main/app/src/assets/i18n/en.json

```diff
@@ -674,6 +674,12 @@
   "zoeken.filter.jaNee.true": "Yes",
   "zoeken.filter.jaNee.false": "No",
   "zoeken.filter.zaak_indicaties": "Case indications",
+  "ZoekVeld.ALLE": "All",
```

Comment:
Moet 'ZoekVeld' niet met een kleine letter beginnen? Alle properties beginnen toch met een kleine letter tenzij ze in hoofdletters zijn gescreven?

Reply:
ja, klopt aangepast